schemas/process.yml

```diff
@@ -61,9 +61,22 @@
         short_override: An array of previous executions for the process, including the initial fork. Only executable and args are set.
         normalize:
           - array
+      - at: process
+        as: responsible
+        short_override: Information about the responsible process.
```

Comment:
Could you add something stating this is an Apple specific concept? I feel like a lot of people won't know what this is, and maybe use it incorrectly if they don't know.